Makefile

```diff
@@ -1,8 +1,8 @@
-PACKAGES=-package lwt.syntax,lwt,dns.lwt,libvirt,cmdliner,ezxmlm,ipaddr,str,conduit,conduit.lwt-unix
+PACKAGES=-package lwt.syntax,lwt,dns.lwt,libvirt,cmdliner,ezxmlm,ipaddr,str,conduit,conduit.lwt-unix,xen-api-client,xen-api-client.lwt,irmin.unix
 INCLUDE=
 OPT=-linkpkg -g 
 OCAMLOPT=ocamlopt -w A-4-44
-FILES=synjitsu.mli synjitsu.ml jitsu.mli jitsu.ml main.ml
+FILES=vm_stop_mode.ml vm_state.ml backends.mli libvirt_backend.ml xapi_backend.ml irmin_backend.mli irmin_backend.ml dns_helpers.ml synjitsu.mli synjitsu.ml jitsu.mli jitsu.ml main.ml
```

Comment:
No, not really - it was just convenient when the project was smaller. A patch would be very welcome :-)